iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Network.java

```diff
@@ -93,6 +98,11 @@
          */
         Stream<Bus> getBusStream();
 
+        /**
+         * Get a Bus.
+         */
+        Bus getBus(String id);
```

Comment:
Same remark here. You can even add the implementation (even it duplicates code).